src/app-layer-ftp.h

```diff
@@ -115,6 +116,7 @@ typedef struct FtpState_ {
     uint8_t *input;
     int32_t input_len;
     uint8_t direction;
+    uint8_t active;
```

Comment:
Could we get rid of direction field in favor of active one ?

Reply:
Good question ... I see that the `direction` member is only used in `FTPDataParse`. What do you think?

Reply:
By reading code, `state->direction` is used for the way of the message. So looks like we need to keep it.